js/adoptMain.js

```diff
@@ -0,0 +1,170 @@
+//  DOG CARD IMAGES  //
+$('.dog-card').hover(function(e) {
+  $('img', this).toggleClass('.image-hover');
```

Comment:
for the class you don't need the period, it's actually adding the class `.image-hover` instead of `image-hover`